messages/src/message.rs

```diff
@@ -108,6 +108,24 @@ pub enum HostRequest {
     /// Request that the modules be reset.
     Reset,
 
+    /// Request that the ResetL line be asserted.
+    AssertReset,
```

Comment:
I can't comment on the actual line because GitHub UI is the worst, but we should bump the protocol version number in this change too.